tests/test_logger.py

```diff
@@ -0,0 +1,88 @@
+# -*- coding: utf-8 -*-
+"""Watchmaker main test module."""
+from __future__ import (absolute_import, division, print_function,
+                        unicode_literals, with_statement)
+
+import logging
+import os
+import shutil
+import sys
+
+from watchmaker.logger import (LOG_LEVELS, exception_hook, make_log_dir,
+                               prepare_logging)
+
+if sys.version_info < (2, 7):
+    import unittest2 as unittest
+else:
+    import unittest
```

Comment:
And `six` at this moment does not support the `assertLogs` assertion.  No doubt lots of people would love that feature.